packages/plugin-core/src/commands/ShowPreviewV2.ts

```diff
@@ -139,28 +133,30 @@ export class ShowPreviewV2Command extends BasicCommand<
   }
 
   async execute(_opts?: CommandOpts) {
-    // Get workspace information
+    const ctx = "ShowPreview";
```

Comment:
`ShowPreviewV2`? - Is the plan to roll this CRA version as the new preview v2; have the current next.JS one become showPreview, and have legacy preview become `Show Preview Legacy`?

Reply:
this is in `ShowPreview`. see new pr